heliopy/util/config.py

```diff
@@ -51,14 +52,12 @@ def load_config():
     config_dict = {}
```

Comment:
Supporting py3.5 means that the `config.read(config_location)` will need `config_location` to be converted to a string before it's passed to `config.read`.

Reply:
Yep! I guess it is more feasible to have both 3.5 and 3.6 until 3.7 gets released.